core/src/types/operator/blocking_operator.rs

```diff
@@ -138,6 +138,13 @@ impl BlockingOperator {
     pub fn info(&self) -> OperatorInfo {
         OperatorInfo::new(self.accessor.info())
     }
+
+    /// Create a new non-blocking operator.
+    ///
+    /// This operation is nearly no cost.
+    pub fn non_blocking(&self) -> Operator {
```

Comment:
Pleae don't touch core's API. We already have `impl From<BlockingOperator> for Operator`